### PR TITLE
Patch 1 - Adding LightFactory and Strand NEO to controller list.

### PR DESCRIPTION
**Pull request by msearancke**

### CODE REVIEW

File: data/controller_data.py [hunk 1]
```diff
@@ -84,6 +84,11 @@
      'image_url': 'http://zero88.com/images/product-pics-scd-server.jpg',
      'tags': ['console']},
   ],
+  0x2239: [
+    {'name': 'LightFactory',
+     'link': 'https://www.lightfactory.net/',
+     'image_url': 'https://www.lightfactory.net/images/lightfactory_square1_small.png'},
+  ],
```
Comment (peternewman) [inline data/controller_data.py]: You probably want to add a tags line with console in it @msearancke so it shows up in this list: http://rdm.openlighting.org/controller/browse?tag=console

File: data/controller_data.py [hunk 2]
```diff
@@ -201,4 +206,9 @@
      'image_url': 'http://www.jese.co.uk/product/image/rdmtri.gif',
      'tags': ['tool', 'RDM Responder Tests', 'USB']},
   ],
+  0x736c: [
+    {'name': 'NEO',
+     'link': 'https://www.strandlighting.com/global/products/neo-console-and-software',
+     'image_url': 'https://www.strandlighting.com/b-dam/strand-lighting/products/neo-console-and-software/NEO-Console-452x410.jpg'},
+  ],
```
Comment (peternewman) [inline data/controller_data.py]: Again

File: data/controller_data.py (hunk 1 above)
Comment (msearancke) [reply data/controller_data.py]: Thanks, done.

File: data/controller_data.py (hunk 2 above)
Comment (msearancke) [reply data/controller_data.py]: done.

File: data/controller_data.py [hunk 3]
```diff
@@ -84,6 +84,12 @@
      'image_url': 'http://zero88.com/images/product-pics-scd-server.jpg',
      'tags': ['console']},
   ],
+  0x2239: [
+    {'name': 'LightFactory',
+     'link': 'https://www.lightfactory.net/',
+     'image_url': 'https://www.lightfactory.net/images/lightfactory_square1_small.png'},
+    'tags': ['console']},
```
Comment (peternewman) [inline data/controller_data.py]: Can you fix the spacing of this please (indent by one).

File: data/controller_data.py [hunk 4]
```diff
@@ -201,4 +207,10 @@
      'image_url': 'http://www.jese.co.uk/product/image/rdmtri.gif',
      'tags': ['tool', 'RDM Responder Tests', 'USB']},
   ],
+  0x736c: [
+    {'name': 'NEO',
+     'link': 'https://www.strandlighting.com/global/products/neo-console-and-software',
+     'image_url': 'https://www.strandlighting.com/b-dam/strand-lighting/products/neo-console-and-software/NEO-Console-452x410.jpg'},
+    'tags': ['console']},
```
Comment (peternewman) [inline data/controller_data.py]: Can you fix the spacing of this please (indent by one).

File: data/controller_data.py (hunk 3 above)
Comment (msearancke) [reply data/controller_data.py]: Done.

File: data/controller_data.py (hunk 4 above)
Comment (msearancke) [reply data/controller_data.py]: Done.

File: data/controller_data.py [hunk 5]
```diff
@@ -84,6 +84,12 @@
      'image_url': 'http://zero88.com/images/product-pics-scd-server.jpg',
      'tags': ['console']},
   ],
+  0x2239: [
+    {'name': 'LightFactory',
+     'link': 'https://www.lightfactory.net/',
+     'image_url': 'https://www.lightfactory.net/images/lightfactory_square1_small.png'},
```
Comment (peternewman) [inline data/controller_data.py]: You've left the old curly bracket at the end of this line:
https://travis-ci.org/OpenLightingProject/rdm-app/jobs/507611257

File: data/controller_data.py [hunk 6]
```diff
@@ -201,4 +207,10 @@
      'image_url': 'http://www.jese.co.uk/product/image/rdmtri.gif',
      'tags': ['tool', 'RDM Responder Tests', 'USB']},
   ],
+  0x736c: [
+    {'name': 'NEO',
+     'link': 'https://www.strandlighting.com/global/products/neo-console-and-software',
+     'image_url': 'https://www.strandlighting.com/b-dam/strand-lighting/products/neo-console-and-software/NEO-Console-452x410.jpg'},
```
Comment (peternewman) [inline data/controller_data.py]: Again

File: data/controller_data.py (hunk 5 above)
Comment (msearancke) [reply data/controller_data.py]: Fixed.

File: data/controller_data.py (hunk 6 above)
Comment (msearancke) [reply data/controller_data.py]: Fixed

**Comment by peternewman** [comment]
Thanks very much @msearancke . We still need to finish our deployment process so these things appear automagically on the website, but there's a PR in progress for that so it's now appeared on my staging site and will get deployed to the live one in due course (either manually or when that PR gets merged):
https://peter-ola-rdm-app.appspot.com/controller/browse?manufacturer=29548
https://peter-ola-rdm-app.appspot.com/controller/browse?manufacturer=8761

We need to rework image grabbing too https://github.com/OpenLightingProject/rdm-app/issues/132 .

**Pull request merged**